k8s/runners/secrets.yaml

```diff
@@ -202,3 +202,73 @@ data:
     xxXxx3XXXXXxXXXxXxxxX3xXXX1XXXXXXxxxXxXxX3XXXXXxXxx3XxxXX09xXxXxXxxXXXXxXX9X
     XXxxxxxXXxXxxXXXxXXXXXxxXXxXXXXxX0XxXXxXx2XXXXXXXxxXXX0XXXx2XxXXXX0xXX1XXxXx
     XXxXXXXXXxxXXxXXXXxxXxxXX0xxXX0xXXx=
+  e4s.gpg:
```

Comment:
Why do the other keys have a `|-` on this line?  According to this [documentation](https://yaml-multiline.info/), it seems like `|` says "keep the newlines", while `-` indicates "no newline at the end".  Do we need that here?

Reply:
Oh wait, it's just documentation, so nevermind.